### DISCUSSION TITLE
VersionSupportedHTTPRequest.gd dictionary error

### DISCUSSION THREAD

**Discussion by ivantea-dev** [Poing-Studios/godot-admob-editor#84] [Q&A] [answered]
Hello! 
So in res://addons/src/core/VersionSupportedHTTPRequest.gd is this line:
var version_support : Dictionary = json.result[AdMobGlobals.get_plugin_version()]
where 
AdMobGlobals.get_plugin_version() 
returns (for me) "v1.3.3".
It checks the version on web: https://gist.githubusercontent.com/gumaciel/ba15bd85ebcc87866c24a819de37cc9a/raw/admob_editor_versions_supported.json
But in the json file there is "1.3.3", not "v1.3.3" which causes an error.

**Comment by gumaciel** [comment, marked as answer]
@ivan-tea135 thank you, i just changed here.

Please verify in few minutes if will solve
